Makefile

```diff
@@ -46,3 +46,25 @@ yq:
 pint_lint:
 	echo "Linting Prometheus rules..."
 	./pint --no-color lint ${EXTRACTED_RULE_FILE}
+
+yamllint:
+	echo "Installing yamllint..."
+	python3 -m pip install pipenv
+	python3 -m pipenv install --dev
+
+.PHONY: yaml_lint
+yaml_lint:
+	echo "Linting YAML files..."
+	python3 -m pipenv run yamllint . && echo "yaml_lint SUCCESS"
```

Comment:
You now have two targets with the same content. You can remove one.

Reply:
You marked this as resolved but you still have both `yaml_lint` and `lint_yamls` doing the same. `yaml_lint` is not being referenced, so I think it should go.

Reply:
Thank you for highlighting it, it has been removed.